packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -16,8 +16,11 @@
  * If ommitted, we will assume _message.
  */
 export const messages = {
+  config_name_text: 'Launch Apex Debugger',
```

Comment:
Added this from hardcoded values, text is similar to the one used by Apex Replay Debugger.